pkg/tools/otelcol-config/remote_control.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	DefaultSumoLogicOpampEndpoint       = "wss://opamp-events.sumologic.com/v1/opamp"
```

Comment:
should it be changed in the opamp_agent.go file? found the endpoint in that file also. 

https://github.com/SumoLogic/sumologic-otel-collector/blob/74a9060c41224a13f8018f8ab511a167d8589bdf/pkg/extension/opampextension/opamp_agent.go#L47

Reply:
Good catch, updated.